src/couch/src/couch_httpd_auth.erl

```diff
@@ -447,7 +447,7 @@ handle_session_req(#httpd{method = 'POST', mochi_req = MochiReq} = Req, AuthModu
             authentication_warning(Req, UserName),
             % clear the session
             Cookie = mochiweb_cookies:cookie(
-                "AuthSession", "", [{path, "/"}] ++ cookie_scheme(Req)
+                "AuthSession", "", [{path, "/"}] ++ cookie_scheme(Req) ++ same_site()
```

Comment:
Should we also add the domain in there? Like we do in: https://github.com/apache/couchdb/blob/c12f0a723cc213679bb90cd263b1187b153627d9/src/couch/src/couch_httpd_auth.erl#L506-L513

I think that might fix issue https://github.com/apache/couchdb/issues/3840

Might even make sense to have a `clear_session(Req)` function to reuse the code and prevent future discrepancies.